src/Microsoft.ML.FastTree/GamTrainer.cs

```diff
@@ -111,22 +111,22 @@ public abstract class ArgumentsBase : LearnerInputBaseWithWeight
         private const string RegisterName = "GamTraining";
 
         //Parameters of training
-        protected readonly TArgs Args;
+        internal readonly TArgs Args;
```

Comment:
>internal [](start = 8, length = 8)

I see eveywhere you changed `protected` to `internal`. Is `private protected` the right modifier for these cases?